### PR TITLE
Meetup 13 - advanced in database/sql

### PR DESCRIPTION
**Pull request by linxGnu**

### CODE REVIEW

**Comment by RuniVN** [comment]
LGMT

File: 13/go-db/code/batchInsert.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import (
+	"database/sql"
+	"fmt"
+	"strconv"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+func main() {
+	db, err := sql.Open("mysql", "root:123@tcp(localhost:3306)/tmp")
```
Comment (huytd) [inline 13/go-db/code/batchInsert.go:12]: Man, you committed database password to GitHub :trollface:

Comment (RuniVN) [reply 13/go-db/code/batchInsert.go:12]: just local stuffs :P

Comment (trietphm) [reply 13/go-db/code/batchInsert.go:12]: @huytd we should remove that password and reset everything to the first commit, right? :trollface:

Comment (huytd) [reply 13/go-db/code/batchInsert.go:12]: Squash everything in the history into just 1 commit :trollface:

**Pull request merged**